mod/discussions/languages/en.php

```diff
@@ -48,24 +48,22 @@
 	'discussion:topic:notify:summary' => 'New discussion topic called %s',
 	'discussion:topic:notify:subject' => 'New discussion topic: %s',
 	'discussion:topic:notify:body' =>
-'%s added a new discussion topic to %s:
```

Comment:
I removed the other changes. I think we should leave the double quotes for clarity.